nupic/research/monitor_mixin/plot.py

```diff
@@ -22,9 +22,13 @@
 """
 Plot class used in monitor mixin framework.
 """
-import matplotlib.pyplot as plt
-import matplotlib.cm as colorModel
+import traceback
 
+try:
+  # We import in here to avoid creating a matplotlib dependency in nupic.
+  import matplotlib.pyplot as plt
+except ImportError:
+  print traceback.format_exc() + "\n"
```

Comment:
Can we print a more human-friendly warning here, saying that plotting won't work or something along those lines?